pkgs/tools/misc/bunnyfetch/default.nix

```diff
@@ -2,25 +2,24 @@
 
 buildGoModule rec {
   pname = "bunnyfetch";
-  version = "unstable-2021-05-24";
+  version = "24370338b936bae1ebdefea73e8372ac0b4d2858";
 
   src = fetchFromGitHub {
-    owner = "Mewyuna";
-    repo = pname;
-    rev = "7bcc45fb590b37f410e60af893e679eb0729ecb1";
-    sha256 = "1lgqrwmxdxd1d99rr0akydfwcsbcmz75fkbq9zrl842rksnp5q3r";
+    owner = "Rosettea";
+    repo = "bunnyfetch";
+    rev = version;
+    sha256 = "09wcffx6ak4djm2lrxq43n27p9qmczng4rf11qpwx3w4w67jvpz9";
```

Comment:
The version should be in the `unstable-yyyy-mm-dd` format, with the date of the commit.

Reply:
Done